library/spdm_requester_lib/libspdm_req_get_capabilities.c

```diff
@@ -107,6 +118,11 @@ static bool validate_responder_capability(uint32_t capabilities_flag, uint8_t ve
             if ((chal_cap == 1) || (key_ex_cap == 1) || (meas_cap == 2) || (mut_auth_cap == 1)) {
                 return false;
             }
+            if (version == SPDM_MESSAGE_VERSION_13) {
```

Comment:
I'd move this down to line 156 with the rest of the 1.3 checks.

Reply:
I add it here purposely.

event_cap relies on other session cap. Most logic can be merged.